crates/subspace-node/src/bin/subspace-node.rs

```diff
@@ -615,33 +581,58 @@ fn main() -> Result<(), Error> {
                         _phantom: Default::default(),
                     };
 
-                    let system_domain_node = domain_service::new_full_system::<
+                    let evm_base_path = BasePath::new(
+                        domain_config
+                            .service_config
+                            .base_path
+                            .config_dir(domain_config.service_config.chain_spec.id()),
+                    );
+
+                    let eth_provider = EthProvider::<
+                        evm_domain_runtime::TransactionConverter,
+                        DefaultEthConfig<
+                            FullClient<
+                                DomainBlock,
+                                evm_domain_runtime::RuntimeApi,
+                                CoreEVMDomainExecutorDispatch,
+                            >,
+                            FullBackend<DomainBlock>,
+                        >,
+                    >::new(
+                        Some(evm_base_path), domain_cli.additional_args()
+                    );
+
+                    let domain_params = domain_service::DomainParams {
+                        domain_id: DomainId::SYSTEM,
```

Comment:
EVM instead of System ? maybe a left over ?

Reply:
Will address it when refactoring `DomainId`, still thinking about that. I have a feeling that we'll probably no longer DomainId::SYSTEM, DomainId::EVM, etc.